src/hotspot/os/aix/porting_aix.cpp

```diff
@@ -244,21 +235,16 @@ bool AixSymbols::get_function_name (
       }
       p_name[i] = '\0';
 
-      // If it is a C++ name, try and demangle it using the Demangle interface (see demangle.h).
-#ifndef DISABLE_DEMANGLE
+      // If it is a C++ name, try and demangle it using the __cxa_demangle interface(see demangle.h).
       if (demangle) {
-        char* rest;
-        Name* const name = Demangle(p_name, rest);
-        if (name) {
-          const char* const demangled_name = name->Text();
-          if (demangled_name) {
-            strncpy(p_name, demangled_name, namelen-1);
-            p_name[namelen-1] = '\0';
+	int status;
+	char *demangled_name = abi::__cxa_demangle(p_name, NULL, NULL, &status);
+        if ((demangled_name != NULL) && (status == 0)) {
```

Comment:
Please change these NULLs to nullptr. There is an ongoing effort to eliminate NULL in the codebase; I believe simply swapping NULL for nullptr is appropriate here.

Reply:
I will modify these NULLs to nullptr. Thank you @backwaterred for pointing out this.